.github/workflows/windows.yaml

```diff
@@ -0,0 +1,54 @@
+name: windows
+on:
+  push:
+    branches: [main]
+    tags: ['v*']
+  pull_request:
+    branches: [main]
+  schedule:
+    - cron: '15 22 * * *'
+  workflow_dispatch: {} # support manual runs
+# Prevent writing to the repository using the CI token.
+# Ref: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#permissions
+permissions: read-all
+jobs:
+  ci:
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        go-version: [1.18.x]
+        include:
+          - go: 1.18.x
+            latest: true
```

Comment:
Since we're not running lint, we don't need this `include` block - I don't think we're referencing `latest` anywhere.